db/structure.sql

```diff
@@ -2,8 +2,8 @@
 -- PostgreSQL database dump
```

Comment:
Should not be committed unless you are adding a migration. Can you remove it from the PR?